js/controllers/guided-tour.js

```diff
@@ -28,6 +28,10 @@ export default class extends Controller {
     if (this.autoplayTimer) window.clearTimeout(this.autoplayTimer);
   }
 
+  currentSlideIndex() {
+    this.slidesTargets.findIndex((x) => x.id === window.location.hash.substring(1));
```

Comment:
Does this need an explicit return?

```suggestion
    return this.slidesTargets.findIndex((x) => x.id === window.location.hash.substring(1));
```

Reply:
You are right. My Ruby brain strikes again.